api/facadeversions.go

```diff
@@ -66,7 +66,7 @@ var facadeVersions = map[string]int{
 	"MachineActions":               1,
 	"MachineManager":               6,
 	"MachineUndertaker":            1,
-	"Machiner":                     3,
+	"Machiner":                     4,
```

Comment:
Usually we don't update a facade version in a dot release.  Do we have approval for this?  Can we delay that part to develop only?

I'm all for removing code, but not sure it's a good enough reason to bump a facade version in a dot release.   And having to deal with potential versioning fall out

Reply:
I think in this case it's ok. It's a controller facade and there will be no issues therefore with version skew. It matters a lot more if it were a client facade, as then the version X CLI would not fully know about a newer X=1 facade in a newer controller.